app/2.0.x/configuration.md

```diff
@@ -651,6 +656,8 @@ This value accepts IPv4, IPv6, and hostnames.
 
 Some suffixes can be specified for each pair:
 
+- `ssl` will require that all connections made through a particular
+  address/port be made with TLS enabled.
```

Comment:
For some reason this line was not present in the generated 2.0.x doc, but it was in the 2.0.x branch of the kong repo. So I am adding it back.